pylint_odoo/checkers/no_modules.py

```diff
@@ -408,6 +414,11 @@ def visit_functiondef(self, node):
                 self.add_message('method-required-super',
                                  node=node, args=(node.name))
 
+        if self.linter.is_message_enabled('old-api7-method-defined'):
+            first_args = [arg.name for arg in node.args.args][:3]
+            if first_args == ['self', 'cr', 'uid']:
```

Comment:
Thanks for feedback.
Do you have a example to change it?

FYI we are using Antony Lesuisse and Fabien Pinckaers' regex
https://twitter.com/fpodoo/status/763480940827938816

They are using `self`: `def .*self.*cr.*uid`